### ISSUE TITLE
Bot keep randomly crashing

### ISSUE THREAD

**Issue by 0wx** [telegraf/telegraf#1709]
My bot keep crashing randomly, tried to debug it on test but it's never fails on test.

this is the error code on production
```
2022-10-01T06:51:46.515767996Z app[web.1]: Error [ERR_HTTP_HEADERS_SENT]: Cannot render headers after they are sent to the client
2022-10-01T06:51:46.515773387Z app[web.1]:     at new NodeError (node:internal/errors:387:5)
2022-10-01T06:51:46.515778488Z app[web.1]:     at ServerResponse.writeHead (node:_http_server:335:13)
2022-10-01T06:51:46.515783650Z app[web.1]:     at /app/node_modules/telegraf/lib/core/network/webhook.js:48:17 {
2022-10-01T06:51:46.515788982Z app[web.1]:   code: 'ERR_HTTP_HEADERS_SENT'
2022-10-01T06:51:46.515794155Z app[web.1]: }
```

The bot never crash on test or when the traffic is low. 
https://github.com/telegraf/telegraf/blob/77fc22740daf8cd9553504ce7e646edfcce3b27e/src/core/network/webhook.ts#L51-L55

This could probably because it catch error when `res.end()` is failing, but still trying do it again, idk.

**Comment by 0wx** [comment]
Thanks @wojpawlik for the fast fix. Anyway when will it published on npm?

**Comment by wojpawlik** [comment]
Hopefully as soon as #1690 is merged.

Install from GitHub in the meantime: `npm install telegraf/telegraf#dbd83ae417dddeae1a335c242541956b5764c79c`.

**Comment by MKRhere** [comment]
Released 4.10.0 including this fix.
